frontend/frontend-react/src/components/template/Header.jsx

```diff
@@ -1,18 +1,28 @@
 import './Header.css'
-import React from 'react'
+import React, { Component } from 'react'
 
-export default props =>
-    <header className="header">
-        <div className="h2 font-weight-bold">Instagram</div>
-        <div className="input-group col-3">
-            <span className="input-group-text"><i className="fa fa-search"></i></span>
-            <input type="text" class="form-control" id="formGroupExampleInput"
-                placeholder={`Buscar`}></input>
-        </div>
-        <div>
-            <i className="icons fa fa-home"></i>
-            <i className="icons fa fa-comments"></i>
-            <i className="icons fa fa-compass"></i>
-            <i className="icons fa fa-heart teste"></i>
-        </div>
-    </header>
\ No newline at end of file
+export default class Main extends Component {
+    render() {  
+        return (
+            <header className="header">
+                <div className="h2 font-weight-bold">Instagram</div>
+                <div className="input-group col-3">
+                    <span className="input-group-text">
+                        <i className="fa fa-search"/>
+                    </span>
+                    <input type="text" class="form-control" id="formGroupExampleInput"
+                        placeholder={`Buscar`}>  
+                    </input>
```

Comment:
You should use `className` instead of `class` in JSX/React and you can make the placeholder simpler:
```suggestion
                    <input type="text" className="form-control" id="formGroupExampleInput"
                        placeholder="Buscar">  
                    </input>
```

Also, you can format it in a way that helps to read the code:
```suggestion
                    <input 
                        type="text" 
                        id="formGroupExampleInput"
                        className="form-control" 
                        placeholder="Buscar"
                     />
```

You can use [Prettier VS Code extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to help you to format the code and make it consistent.